### ISSUE TITLE
Does it support "xls" file parsing?

### ISSUE THREAD

**Issue by vela666** [zurmokeeper/excelize#28]

**Comment by zurmokeeper** [comment]
@vela666 

Yes there is this plan, but it is a bit difficult, originally I wanted to xls format with` xlsx `this library to parse, and then the data structure of their own organized into the same as now, but the implementation of the workload is quite a lot, if you start from scratch, although you can refer to `the source code of xlsx`, but one person to do is really difficult, if there are more people together or more to give me some encouragement and support may be able to accelerate the word, now I'm a bit difficult, sorry.
